pkg/operator/reportingutil/util.go

```diff
@@ -35,8 +35,11 @@ func AWSBillingPeriodTimestamp(date time.Time) string {
 	return date.Format(AWSUsagePartitionDateStringLayout)
 }
 
-func FullyQualifiedTableName(prestoTable *cbTypes.PrestoTable) string {
-	return presto.FullyQuaifiedTableName(prestoTable.Status.Catalog, prestoTable.Status.Schema, prestoTable.Status.TableName)
+func FullyQualifiedTableName(prestoTable *cbTypes.PrestoTable) (string, error) {
+	if len(prestoTable.Status.Catalog) == 0 || len(prestoTable.Status.Schema) == 0 || len(prestoTable.Status.TableName) == 0 {
```

Comment:
Instead of a single if, use 3 for each case, and give a more specific error message for each missing field. You can even accumulate errors into a slice, and if it's non-empty, join the errors together into one.

eg:
```
...
var errs []string
if len(prestoTable.Status.Catalog) == 0 {
  errs = append(errs, "catalog is unset")
}
if len(prestoTable.Status.Schema) == 0 {
  errs = append(errs, "schema is unset")
}
...

if len(errs) != 0) {
  return fmt.Errorf("PrestoTable status is invalid: %s", strings.Join(errs, ", "))
}
...